src/components/web-package-version.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+const navStyles = {
+  fontSize: '0.5em',
+  letterSpacing: '0.2em',
+};
+
+const WebPackageVersion = () => (
+  <sup>
+    <span className="badge badge-pill badge-secondary text-uppercase" style={navStyles}>
+      alpha {process.env.REACT_APP_VERSION}
```

Comment:
use `process.env.npm_package_version` which is a standard node env variable loaded from `package.json`